package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cht-conf-test-harness",
-  "version": "1.0.0",
```

Comment:
Weren't we on 4.0.0?
https://github.com/medic/cht-conf-test-harness/commit/12e171a74074f9565bdcc04b86d707590accfaf8#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L3

Reply:
Sorry you are right, I took the last from npm.

Thanks its now fixed